js/packages/cli/src/cli-nft.ts

```diff
@@ -66,18 +68,32 @@ programCommand('mint')
     if (toWallet) {
       receivingWallet = new PublicKey(toWallet);
     }
-    await mintNFT(
-      solConnection,
-      walletKeyPair,
-      url,
-      true,
-      collectionKey,
-      supply,
-      verifyCreators,
-      structuredUseMethod,
-      receivingWallet,
-    );
-  });
+    if (MintMultiple !== undefined) MintMultiple = 1;
+    const NUMBER_OF_NFTS_TO_MINT = parseInt(MintMultiple, 10);
```

Comment:
First name convention... second this parse should be done before any validation like (MintMultiple > maxSupply). I believe   the library should parse things for you already (need to double check that)